src/main/java/chylex/customwindowtitle/mixin/DisableVanillaTitle.java

```diff
@@ -8,8 +10,17 @@
 
 @Mixin(Minecraft.class)
 public final class DisableVanillaTitle {
+
 	@Inject(method = "updateTitle()V", at = @At("HEAD"), cancellable = true)
 	private void updateTitle(final CallbackInfo info) {
 		info.cancel();
 	}
+
+	@Inject(method = "onResourceLoadFinished", at = @At("HEAD"))
+	private void onFinishedLoading(final CallbackInfo callbackInfo) {
+		final TitleConfig config = TitleConfig.getInstance();
```

Comment:
This mixin should be extracted into a separate file, otherwise the enclosing class name "DisableVanillaTitle" no longer matches what the class does.